docs/reference/method/MongoDBGridFSBucket-downloadToStreamByName.txt

```diff
@@ -0,0 +1,40 @@
+===========================================
+MongoDB\\GridFS\\Bucket::downloadToStream()
+===========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\GridFS\\Bucket::downloadToStreamByName()
+
+   Selects a GridFS file by its ``filename`` and copies its contents to a
+   writable stream.
+
+   .. code-block:: php
+
+      function downloadToStreamByName($filename, $destination, array $options = []): void
+
+   :phpmethod:`MongoDB\\GridFS\\Bucket::downloadToStreamByName()` has the
+   following parameters:
+
+   .. include:: /includes/apiargs/MongoDBGridFSBucket-method-downloadToStreamByName-param.rst
+
+   The ``$options`` parameter supports the following options:
+
+   .. include:: /includes/apiargs/MongoDBGridFSBucket-method-downloadToStreamByName-option.rst
+
+.. todo: add examples
```

Comment:
Does `.. todo` actually render any output or is there merely an internal comment that the docs team can use to find areas needing improvement down the line?